tests/test_process.py

```diff
@@ -504,34 +504,17 @@ def test_process_folder(
     )
     sampling_rate = 8000
     path = str(tmpdir.mkdir('wav'))
-    files = [f'{path}/file{n}.wav' for n in range(num_files)]
-    rel_path = os.path.relpath(path)
```

Comment:
For some reasons `os.path.relpath()` was not working in the Windows job (on my VM it did...). But I anyway find this test a bit messy and hard to read. I know simply compare if `process_folder()` gives the same result as `process_files()` on all files in the folder. Which is essentially what we want.

Reply:
Yes, you are right, and it simplifies the test code a lot.